### ISSUE TITLE
Newest release cause error

### ISSUE THREAD

**Issue by aghajani** [flant/curator-opensearch#13]
Running the new release causes this error. I guess it's because of some version updates:

```log
{"@timestamp": "2022-11-09T13:06:43.941Z", "function": "run", "linenum": 163, "loglevel": "INFO", "message": "Creating client object and testing connection", "name": "curator.cli"}
Traceback (most recent call last):
  File "/usr/local/bin/curator", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 829, in __call__
{'hosts': ['XXX'], 'port': 443, 'use_ssl': True, 'ssl_no_validate': False, 'ssl_show_warn': False, 'url_prefix': '', 'aws_sign_request': False, 'client_key': None, 'aws_secret_key': None, 'aws_key': None, 'client_cert': None, 'http_auth': None, 'aws_token': None, 'certificate': None, 'timeout': 21600, 'connection_class': <class 'opensearchpy.connection.http_requests.RequestsHttpConnection'>, 'verify_certs': True, 'ca_certs': '/usr/local/lib/python3.10/site-packages/certifi/cacert.pem', 'aws_region': False, 'api_key': None}
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/curator/cli.py", line 242, in cli
    run(config, action_file, dry_run)
  File "/usr/local/lib/python3.10/site-packages/curator/cli.py", line 165, in run
    client = get_client(**client_args)
  File "/usr/local/lib/python3.10/site-packages/curator/utils.py", line 1053, in get_client
    LOGGER.info("kwargs = {0}".format(password_filter_special(kwargs.copy())))
  File "/usr/local/lib/python3.10/site-packages/curator/utils.py", line 2132, in password_filter_special
    return iterdict(data)
  File "/usr/local/lib/python3.10/site-packages/curator/utils.py", line 2128, in iterdict
    mydict.update({"http_auth": (value[0],"REDACTED")})
TypeError: 'NoneType' object is not subscriptable
```

**Comment by uzhinskiy** [comment]
Hi @aghajani ! Thank you for your report.
Please provide config.yml and action.yml for reproduce this issue.

**Comment by uzhinskiy** [comment]
@aghajani : Be noticed, what `http_auth` option is depricated and you must use `username` & `password` instead.

**Comment by uzhinskiy** [comment]
Fixed in v0.0.7
